app/views/errorDetail.html

```diff
@@ -1,8 +1,35 @@
 <h2><label for="editId">{{editType | capitalize}} Edit ID:</label> <select id="editId" ng-model="selectedEditId" ng-change="goToEditDetail()" ng-options="id for id in siblingEdits"></select></h2>
 <p>{{editError.explanation}}</p>
 
-<p>Please review and correct all of the errors listed below in your system of record. Once corrected, re-select the updated file to validate the data.</p>
+<p ng-if="['syntactical', 'validity', 'macro'].indexOf(editType) !== -1">Please review and correct all of the errors listed below in your system of record. Once corrected, re-select the updated file to validate the data.</p>
```

Comment:
Yes, because only the quality edits require that you navigate all the way through all of the errors "verifying" each one which is what the instructions on L5 tell you.